console/client/src/layout/IDELayout.tsx

```diff
@@ -165,8 +202,17 @@ export function IDELayout() {
           </Tab.Group>
         </div>
         <SidePanel />
-        <RequestModal />
       </div>
+      {invalidTabMessage && (
+        <Notification
+          color='text-red-400'
+          icon={
+            <InformationCircleIcon className='flex-shrink-0 inline w-4 h-4 mr-3' />
+          }
+          title='Info Alert!'
```

Comment:
Should this just be `Alert` vs. `Info Alert`? Or is the intended message format?

<img width="336" alt="Screenshot 2023-08-28 at 1 01 51 PM" src="https://github.com/TBD54566975/ftl/assets/51647/fc5dc470-cca4-40bd-bd09-51b7f5e163c9">

Reply:
Let's go with alert